### ISSUE TITLE
Add standard tests

### ISSUE THREAD

**Issue by alex-konovalov** [gap-packages/AutoDoc#152]
There is no `TestFile` component in the `PackageInfo.g` file. It is recommended to specify a short test (to run for no more than several minutes) which may be used to check that a package works as expected. For deposited packages, these tests are run regularly as a part of the standard GAP test suite. Please see (in GAP 4.9.0 or GAP development version) `?TestPackage`, `?Tests files for a GAP package` and also `?TestDirectory` for more information.

**Comment by fingolfin** [comment]
Adding a meaningful test suite for AutoDoc is non-trivial, and `TestDirectory` is useless for this.

**Comment by fingolfin** [comment]
See also #57 and #123

**Comment by alex-konovalov** [comment]
`TestDirectory` is not the only way to test it of course. You can test something and analyse how did it go, and at the end do
```
if testresult then
  Print("#I  No errors detected while testing\n");
  QUIT_GAP(0);
else
  Print("#I  Errors detected while testing\n");
  QUIT_GAP(1);
fi;
```
like e.g. in https://github.com/gap-packages/qaos/blob/master/tst/testall.g

But indeed doing this in some meaningful way for Autodoc is tricky and requires some time to implement.

On a related note, the standard tests need not to be a test used in Travis. The same QaOS package now has a simple offline test specified as standard in PackageInfo.g and extended test for Travis. Perhaps #123 may also do something similar - just building the manual of AutoDoc itself, maybe couple of other packages, and collect code coverage statistics.

**Comment by fingolfin** [comment]
Resolved by PR #183 and 7c90b2cee1c995797ac02a65c4c6aabe104769cb
